app/components/base/buttons/clickable/clickable.component.ts

```diff
@@ -1,18 +1,58 @@
-import { Component, EventEmitter, HostBinding, HostListener, Input, Output } from "@angular/core";
+import {
+    Component, EventEmitter, HostBinding, HostListener,
+    Input, OnChanges, OnDestroy, Output, SimpleChanges,
+} from "@angular/core";
 
+import { AuthorizationHttpService, Permission } from "app/services";
+import { Subscription } from "rxjs";
 import "./clickable.scss";
 
 @Component({
     selector: "bl-clickable",
     template: `<ng-content></ng-content>`,
 })
-export class ClickableComponent {
+export class ClickableComponent implements OnChanges, OnDestroy {
     @Input() @HostBinding("class.disabled") public disabled = false;
-
+    @Input() public permission?: Permission;
     @Output() public do = new EventEmitter<Event>();
-
     @HostBinding("tabindex") public tabindex = "0";
     @HostBinding("class.focus-outline") public focusOutline = true;
+    public subtitle = "";
+
+    private _sub: Subscription;
+
+    constructor(private authHttpService: AuthorizationHttpService) {
+    }
+
+    public ngOnChanges(changes: SimpleChanges): void {
+        if ("disabled" in changes) {
+            this.tabindex = this.disabled ? "-1" : "0";
```

Comment:
how about you just move this at the end of ngChanges and remove the if statement so it computes it everytime and you can remove the reasignment below